shared/menubar/remote-proxy.desktop.js

```diff
@@ -87,8 +83,9 @@ const RenderExternalWindowBranch = (ComposedComponent: React.ComponentType<any>)
   }
 
 // Actions are handled by remote-container
-export default connect<Props | {}, _, _, _, _>(
+export default namedConnect<Props | {}, _, _, _, _>(
   mapStateToProps,
   () => ({}),
-  mergeProps
+  mergeProps,
+  'MenubarRemoteProxy'
```

Comment:
just so its more clear in the react tree what this is, else its just "PureComponent"